frankenphp.c

```diff
@@ -55,6 +55,7 @@ typedef struct frankenphp_server_context {
 	uintptr_t current_request;
 	uintptr_t main_request;				/* Only available during worker initialization */
 	char *cookie_data;
+	bool finished;
```

Comment:
We need some way to track whether the request is "finished" (i.e., no more output and output flushed) or not.